### PR TITLE
ERMS-2146: Bulk Prozesse bei Teilnehmerverwaltung ermöglichen

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -2007,6 +2023,8 @@ subscription.linkPackagesMembers.unlinkInfo.withIE.confirm = Do you really want
 subscription.linkPackagesMembers.unlinkInfo.onlyPackage.confirm = Do you really want to unlink all packages and keep issue entitlements for all licenses? The process can not be undone afterwards!
 subscription.linkPackagesMembers.unlinkInfo.withIE.successful = The package unlink was successful and all issue entitlements were deleted!
 subscription.linkPackagesMembers.unlinkInfo.onlyPackage.successful = The package unlink was successful. The issue entitlements can still be seen in holding!
+subscription.linkPackagesMembers.unlinkInfo.withIE.fail = The package decoupling was successful and all issue entitlements was deleted!
```
Comment (croqueGrec09) [inline app/grails-app/i18n/messages.properties]: unlinking - should we not be consistent?

File: app/grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -1862,6 +1862,13 @@ subscription.details.unlink.pendingChanges = Pending Changes
 subscription.details.unlink.pendingChanges.numbers = Number of IEs: 
 subscription.details.unlink.pendingChanges.numbers.action = Pending Changes will be deleted
 subscription.details.unlink.successfully = The disassociation was successful
+subscription.details.unlink.notSuccessfully = Die Entknüpfung war nicht erfolgreich
```
Comment (croqueGrec09) [inline app/grails-app/i18n/messages.properties]: translation?

File: app/grails-app/views/subscription/linkLicenseMembers.gsp [hunk 3]
```diff
@@ -34,54 +37,74 @@
         id="${parentSub.id}">${parentSub.name}</g:link><br><br>
 
     <g:if test="${parentLicense}">
-        <g:message code="subscription.linkLicenseMembers.license" args="${args.superOrgType}"/>: <g:link controller="license"
-                                                                             action="show"
-                                                                             id="${parentLicense?.id}">${parentLicense?.reference}</g:link>
+        <g:message code="subscription.linkLicenseMembers.license" args="${args.superOrgType}"/>: <g:link
+            controller="license"
+            action="show"
+            id="${parentLicense?.id}">${parentLicense?.reference}</g:link>
```
Comment (croqueGrec09) [inline app/grails-app/views/subscription/linkLicenseMembers.gsp]: Why the null checks? We did that already with the if-block!

File: app/grails-app/views/subscription/linkPackagesMembers.gsp [hunk 4]
```diff
@@ -62,229 +76,255 @@
...
 
         <div class="ui buttons">
-            <g:link class="ui button js-open-confirm-modal"
+            <g:link class="ui button negative js-open-confirm-modal"
                     data-confirm-tokenMsg="${message(code: 'subscription.linkPackagesMembers.unlinkInfo.onlyPackage.confirm')}"
                     data-confirm-term-how="ok" action="processUnLinkPackagesConsortia" id="${params.id}"
                     params="[withIE: false]">${message(code: 'subscription.linkPackagesMembers.unlinkInfo.onlyPackage')}</g:link>
             <div class="or"></div>
-            <g:link class="ui button js-open-confirm-modal"
+            <g:link class="ui button negative js-open-confirm-modal"
```
Comment (croqueGrec09) [inline app/grails-app/views/subscription/linkPackagesMembers.gsp]: Beware of cost items linked to IssueEntitlements!

File: app/grails-app/views/subscription/subscriptionPropertiesMembers.gsp [hunk 5]
```diff
@@ -314,7 +326,7 @@
                                 <tr>
                                     <th>${message(code: 'sidewide.number')}</th>
                                     <th>${message(code: 'default.sortname.label')}</th>
-                                    <th>${message(code: 'subscriptionDetails.members.members')}</th>
+                                    subscription.propertiesMembers.subscriber                             <th>${message(code: 'subscriptionDetails.members.members')}</th>
```
Comment (croqueGrec09) [inline app/grails-app/views/subscription/subscriptionPropertiesMembers.gsp]: translation string without call to translation file

File: app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy [hunk 6]
```diff
@@ -22,6 +22,7 @@ import grails.converters.JSON
 import grails.doc.internal.StringEscapeCategory
 import grails.plugin.springsecurity.annotation.Secured
 import groovy.time.TimeCategory
+import org.apache.lucene.index.DocIDMerger
```
[the remark this answers is not in the document]
Comment (Moetez16) [reply app/grails-app/controllers/com/k_int/kbplus/SubscriptionController.groovy]: Not from me

**Comment by croqueGrec09** [comment]
There are conflicts I am aware to merge ...

**Pull request merged**